feedback/lib/src/feedback_builder/string_feedback.dart

```diff
@@ -89,9 +89,10 @@ class _StringFeedbackState extends State<StringFeedback> {
         ),
         TextButton(
           key: const Key('submit_feedback_button'),
-          child: Text(
-            FeedbackLocalizations.of(context).submitButtonText,
-          ),
+          child: Text(FeedbackLocalizations.of(context).submitButtonText,
+              style: TextStyle(
+                color: FeedbackTheme.of(context).activeFeedbackModeColor,
+              )),
```

Comment:
```suggestion
              ),),
```

Reply:
Basically, I am trying to fix https://github.com/ueman/feedback/issues/245 which will allow Submit button Color to be configurable